lib/luadata.c

```diff
@@ -34,6 +37,12 @@ static inline luadata_t *luadata_checkdata(lua_State *L, lua_Integer *offset, lu
 
 #define luadata_checkint(L, offset, T)	luadata_checkdata((L), &(offset), sizeof(T##_t))
 
+static inline void luadata_editablechecker(lua_State *L, luadata_t *data)
```

Comment:
I would use `*CHECKER` for the template and `check*` for the function name..
```suggestion
static inline void luadata_checkeditable(lua_State *L, luadata_t *data)
```